### PR TITLE
Create management command

### PR DESCRIPTION
**Pull request by Camelia-Br**
https://roverdotcom.atlassian.net/jira/software/c/projects/DEV/boards/402?modal=detail&selectedIssue=DEV-82838&assignee=618b962e86c210006a92ca33

### CODE REVIEW

File: stays/management/commands/recoveryimport.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                    f"The imported file contains {rows_with_errors} errors"
+                )
+            )
+        else:
+            self.stdout.write(
+                self.style.SUCCESS("The command has been executed successfully!")
+                )
+ 
+    @transaction.atomic
+    def get_data(self, row):
```
Comment (luizacoman) [inline stays/management/commands/recoveryimport.py]: rather rename it to get_or_create_data as it is a little confusing to add transaction on a "get" function

File: stays/management/commands/recoveryimport.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        for csv_file in options['csv_file']:
+            dataReader = csv.DictReader(open(csv_file))
+            for row in dataReader:
+                try:
+                    self.get_data(row)
+                    imported_rows += 1
+                except Exception as e:
+                    rows_with_errors += 1
+                    self.stdout.write(self.style.ERROR(e))
+                    continue
```
Comment (luizacoman) [inline stays/management/commands/recoveryimport.py]: the continue here is not necessary as you are at the end of the for. If you want to continue from the outter for, assign a variable and check it outside this for

**Comment by luizacoman** [comment]
Great work, as a brief summary on what's different in production:
- all the management commands that will be running in production will start with prod_
- the management commands will held in the help text a brief example on how to run the command, as we are using git action to run them. eg. 'Usage: {"command": "prod_command", "args": ["<opk>"]}'
- all commands will have a subconcept added
- commands will be tested using CommandTestCase

**Review by tintesancristiana — COMMENTED** [review]
This comment is optional because it is not related to this pr. You can use black to format your code. I saw that is installed in requirements.txt file, so you can use it by running the command black stays - this will format all the files from the stays app or you can use it to format just one file by running black stays/file_name.py. Also, you can use isort to group and sort the imports. You can add isort in your requirements.txt file , run the build command and then just run the same commands: isort stays or isort stays/file_name.,py. Black and isort are also used in the Rover project to format the code.

**Review by tintesancristiana — APPROVED** [review]

**Review by luizacoman — APPROVED** [review]

**Pull request merged**